### ISSUE TITLE
Data tab is empty with Ember Data v2.12.0

### ISSUE THREAD

**Issue by josemarluedke** [emberjs/ember-inspector#642]
Using Ember Data v2.12.0, there is no model types listened. 

![screen shot 2017-03-15 at 12 58 04 pm](https://cloud.githubusercontent.com/assets/230476/23968234/2a3da720-097f-11e7-9128-3343ab4e0618.png)

![screen shot 2017-03-15 at 12 58 24 pm](https://cloud.githubusercontent.com/assets/230476/23968324/811ef38c-097f-11e7-9573-350ecdb46ea3.png)

**Comment by rwjblue** [comment]
What version of ember?

**Comment by johnunclesam** [comment]
The problem here is ember-resolver v3.0.0.

**Comment by josemarluedke** [comment]
Yep, I confirmed that with an older version of ember-resolver it works fine. Thanks @johnunclesam for pointing that out.

**Comment by rwjblue** [comment]
@stefanpenner - Seems like we should roll back those changes in ember-resolver?

**Comment by teddyzeenny** [comment]
@rwjblue what are the changes?

**Comment by stefanpenner** [comment]
> @stefanpenner - Seems like we should roll back those changes in ember-resolver?

Strange, I totally tested this. Let me take a look this afternoon and fix.

**Comment by DamianGaleano9** [comment]
I have the same issue, right now I'm using ember resolver 2.1.1 , I can't see my Data 
![image](https://cloud.githubusercontent.com/assets/16900365/24172948/26a9e32a-0e58-11e7-9c0b-5354adaef882.png)

![image](https://cloud.githubusercontent.com/assets/16900365/24172975/38c5454a-0e58-11e7-807b-1c3c521ef075.png)

**Comment by PoohOka** [comment]
Not only the data, but also all routes. The follow warnings happen when clicking on Routes tab in the inspector.

<img width="629" alt="screen shot 2017-03-22 at 3 10 05 pm" src="https://cloud.githubusercontent.com/assets/1761107/24187937/e971afe6-0f11-11e7-977f-dbcebdb3866b.png">
<img width="298" alt="screen shot 2017-03-22 at 3 10 31 pm" src="https://cloud.githubusercontent.com/assets/1761107/24187939/ed0e2882-0f11-11e7-88f6-392b94054b78.png">

ember-resolver: 3.0.0

**Comment by curtisgr** [comment]
For those that find this issue before it is fixed, dropping down to the following works:

ember-resolver@2.1.1
ember-source@2.11.3
ember-inspector@2.0.4

**Comment by dzbo** [comment]
@curtisgr also work with latest ember-source 2.12.0

The only problem like said above is with latest ember-resolver 3.0.0
It works in this setup:

```
ember-resolver@2.1.1
ember-source@2.12.0
ember-inspector@2.0.5
```

**Comment by josemarluedke** [comment]
This issue has been solved with `ember-resolver@3.0.1`. For reference, see issue [here](ember-cli/ember-resolver#175).
